optuna/visualization/parallel_coordinate.py

```diff
@@ -7,21 +7,17 @@
 from optuna.visualization.utils import _check_plotly_availability
 from optuna.visualization.utils import is_available
 
-if type_checking.TYPE_CHECKING:
-    from typing import DefaultDict  # NOQA
-    from typing import List  # NOQA
-    from typing import Optional  # NOQA
+from typing import DefaultDict  # NOQA
+from typing import List  # NOQA
+from typing import Optional  # NOQA
+from optuna.study import Study  # NOQA
```

Comment:
I guess you can remove these `# NOQA` comments (except for `DefaultDict` because it's used at comment-based type annotation).

```suggestion
from typing import List
from typing import Optional
from optuna.study import Study
```